app/services/food_service.rb

```diff
@@ -0,0 +1,13 @@
+require 'faraday'
+class FoodService
+  def self.connection
+    Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/foods/") do |f|
+      f.params['api_key'] = 'Ze3akdRIKt8TZiohIkLEfF5va0Q5WHAshPSFaHkl'
+    end
+  end 
+
+  def self.all_food
+    response = connection.get("list")
```

Comment:
We should be passing the user's query param to this method from the controller and our argument here should be the rest of the path, so something like:
`response = connection.get("foods/search?query=#{query}")`